perian/perian-core/src/utils/plugin.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import type { Plugin } from '../../types/types.js'
+import { addNetwork } from '../commons/network.js'
+
+export const addPlugin = (moduleName: Plugin) => {
+  if (Array.isArray(moduleName)) {
+    moduleName.forEach(addPlugin)
+  } else {
+    try {
+      if (!moduleName.startsWith('@perian/')) {
```

Comment:
No, this defeats the whole point, since only you control the '@perian' namespace.

Reply:
this whole thing was removed